app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class SubmissionsController < ApplicationController
+  load_and_authorize_resource
+
+  def create
+    binding.pry
```

Comment:
also we need to remove this from the code! not good to put a binding.pry into production...